l/libnsl/stone.yaml

```diff
@@ -0,0 +1,25 @@
+name        : libnsl
+version     : 2.0.1
+release     : 1
+summary     : Public client interface library for NIS(YP)
+license     : LGPL-2.1-only
+homepage    : https://github.com/thkukuk/libnsl
+description : |
+    Public client interface library for NIS(YP) and NIS+
+
+upstreams:
+  - https://github.com/thkukuk/libnsl/releases/download/v2.0.1/libnsl-2.0.1.tar.xz: 5c9e470b232a7acd3433491ac5221b4832f0c71318618dc6aa04dd05ffcd8fd9
+builddeps   :
+    - libtirpc-devel
+rundeps     :
+    - glibc
+    - libtirpc
+environment : |
+    export MAKE_ARGS="--disable-static"
+setup       : |
+    %configure $MAKE_ARGS
```

Comment:
Since MAKE_ARGS is only used as an argument to configure you can just do
```
setup       : |
    %configure \
        --disable-static
```
No environment key necessary here.